### PR TITLE
Binary panel component

### PR DESCRIPTION
**Pull request by daniellemullis**
- extended panel style to tables
- extended panel to 2 column layout for displaying binary optons

### CODE REVIEW

**Comment by daniellemullis** [comment]
Extended the panel pattern to display lists of content side by side for easier comparison
panel-binary

**Comment by daniellemullis** [comment]
Extended panel pattern to work with tables to similar effect
table-panel

**Comment by daniellemullis** [comment]
I think the tabular one is easier to read when doing a side by side comparison. However, this might not always work eg. do's and don'ts won't always have an opposite.

**Comment by daniellemullis** [comment]
Example of dos and don't – this can be combined with the check/cross lists.
panel-dos-donts

**Comment by daniellemullis** [comment]
Reduced the default padding on desktop just in this variation

File: views/mouth-ulcers/main-content.md [hunk 1]
```diff
@@ -6,22 +6,37 @@ Mouth ulcers need time to heal and there’s no quick fix. Avoiding things that
...
+        <li>eat a healthy, balanced diet</li>
+      </ul>
+    </div>
+  </article>
+  <article class="panel__column">
+    <div class="panel__header">
+      <h3>Don't</h3>
+    </div>
+    <div class="panel__content">
+      <ul class="list-check">
```
Comment (teneightfive) [inline views/mouth-ulcers/main-content.md]: You should be able to use markdown for this content and use the [attribute addon](https://www.npmjs.com/package/markdown-it-attrs) for adding the classname

File: assets/css/nhsuk.css [hunk 2]
```diff
@@ -740,54 +740,64 @@ table {
...
+  padding: 16px 16px 16px 0;
   vertical-align: top;
 }
 
 @media (min-width: 641px) {
   table th,
   table td {
-    font-size: 16px;
-    line-height: 24px;
+    font-size: 20px;
```
Comment (teneightfive) [inline assets/css/nhsuk.css:767]: If you're using the same combination as the 20px font size we're already using you can use this sass mixin to set font size and line-height: `@include core-font(20)`

It also add the ability to change the font size at different resolutions.

File: views/mouth-ulcers/main-content.md (hunk 1 above)
Comment (daniellemullis) [reply views/mouth-ulcers/main-content.md]: Not sure whether you mean the wrapper `div` or the `ul`? 

Can you add `<div>` in markdown? If not, anything wrapped in a `<div>` is treated as html. So the `ul` renders

```
- list item
- list item
- list item
- etc
```

What am I missing?

File: assets/stylesheets/units/_tables.scss [hunk 3]
```diff
@@ -6,14 +6,24 @@ table {
...
-    padding: ($baseline-grid-unit * 3) ($baseline-grid-unit * 3) ($baseline-grid-unit * 3) 0;
+    @include core-font(20);
+    border-bottom: 2px solid $grey-3;
+    padding: ($baseline-grid-unit * 4) ($baseline-grid-unit * 4) ($baseline-grid-unit * 4) 0;
     vertical-align: top;
   }
 
   th {
-    @include bold-font(16);
+    @include bold-font(20);
```
Comment (daniellemullis) [inline assets/stylesheets/units/_tables.scss:16]: That's what I did.

File: views/mouth-ulcers/main-content.md (hunk 1 above)
Comment (daniellemullis) [reply views/mouth-ulcers/main-content.md]: Also aiming to have a maximum of 3 markdown files for each condition to make it easier if we decide to delegate content updates

**Pull request merged**